substrate/frame/contracts/src/benchmarking/mod.rs

```diff
@@ -2582,19 +2582,24 @@ benchmarks! {
 		let origin = RawOrigin::Signed(instance.caller.clone());
 	}: call(origin, instance.addr, 0u32.into(), Weight::MAX, None, vec![])
 
-	// We make the assumption that pushing a constant and dropping a value takes roughly
-	// the same amount of time. We call this weight `w_base`.
-	// The weight that would result from the respective benchmark we call: `w_bench`.
+	// We add two `i64` integers from local variables and store the result into yet another
+	// local variable. The combination of this computation is our weight base `w_base`.
 	//
-	// w_base = w_i{32,64}const = w_drop = w_bench / 2
+	// The weight that would result from the respective benchmark we call: `w_bench`.
 	#[pov_mode = Ignored]
-	instr_i64const {
+	instr_i64add {
```

Comment:
if we change things here, we also need to update
 
https://github.com/paritytech/polkadot-sdk/blob/6c3c1b6f26ea14daed1140110ff057776f8ec9c1/substrate/frame/contracts/src/schedule.rs#L398-L404

Reply:
Ah thanks, I was already wondering about this!

Reply:
yes, let's see if that makes CI happy 🤞